configuration/globalproperties/additional_clinical_gp.xml

```diff
@@ -0,0 +1,29 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>emrapi.sqlSearch.noVitalsPatients</property>
```

Comment:
Let's rename `emrapi.sqlsearch.awaitingVitals`

Reply:
@rbuisson great. I'll update this. 
What would you suggest as the message translations for the patient list tab then?

> Awaiting vitals / En attente de signes vitaux

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/6/commits/87ec2f0ccc4b2289023e264352aaec7879b60a36)